### PR TITLE
updated the script commands

### LINKED ISSUE #938: [solidstart-tanstack-tailwind] Add `serve` script, currently missing
# Description
- there is currently no `serve` script in the package.json file. This is referenced in the README and should be fine once it's been added

### PR DESCRIPTION
**Pull request by AllStackDev1**
## Type of change

- [x] Documentation change
- [ ] Bug fix

## Checklist

- [ ] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [ ] This starter kit has been approved by the maintainers
- [ ] Changes for this new starter kit are being pushed to an integration branch instead of main
- [ ] All dependencies are version locked
- [x] This fix resolves #938
- [ ] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/solidstart-tanstackquery-tailwind-modules/README.md [hunk 1]
```diff
@@ -71,11 +71,11 @@ git clone https://github.com/thisdot/starter.dev.git
 
 ## Commands
 
-- `npm run dev` - Runs the development server on localhost port 3000 with HMR
+- `npm run dev` - starts a development server backed by Vite on localhost port 3000 with HMR
```
Comment (Megio) [inline starters/solidstart-tanstackquery-tailwind-modules/README.md:74]: Miss the capitol letter for "start" in the command description

**Review by Megio — COMMENTED** [review]
Everything is ok, but in this section the commands should be "pure" (without any package manager). 
FE:
`dev`- starts a development server backed by Vite on localhost port 3000 with HMR
`start`- Starts the production build with a local version of adapter.

And so on...

File: starters/solidstart-tanstackquery-tailwind-modules/README.md (hunk 1 above)
Comment (hdJerry) [reply starters/solidstart-tanstackquery-tailwind-modules/README.md:74]: the port number is wrong also.

Comment (AllStackDev1) [reply starters/solidstart-tanstackquery-tailwind-modules/README.md]: 👍🏾

**Review by Megio — APPROVED** [review]
LGTM

File: starters/solidstart-tanstackquery-tailwind-modules/README.md [hunk 2]
```diff
@@ -71,14 +71,14 @@ git clone https://github.com/thisdot/starter.dev.git
...
-- `npm run test` - Runs the test suite
-- `npm run storbook` - To showcase the component library
-- `npm run build` - Builds a production version of the app to deploy
-- `npm run serve` - Serves a production build on localhost port 4173
-- `npm run lint` - Uses eslint to find potential issues in the codebase
-- `npm run lint-fix` - Tries to auto-fix potential issues
-- `npm run format` - Fixes formatting issues in the codebase
+- `dev` - Starts a development server backed by Vite on localhost port 5173 with HMR
+- `start` - Starts the production build with a local version of adapter.
+- `build` - Builds a production version of the app to deploy
```
Comment (hdJerry) [inline starters/solidstart-tanstackquery-tailwind-modules/README.md:76]: it's nice if we arrange them accordingly. build before start.

**Review by hdJerry — CHANGES_REQUESTED** [review]

File: starters/solidstart-tanstackquery-tailwind-modules/README.md (hunk 2 above)
Comment (AllStackDev1) [reply starters/solidstart-tanstackquery-tailwind-modules/README.md:76]: Resolved in #938

**Pull request merged**